### ISSUE TITLE
How to paste from clipboard?

### ISSUE THREAD

**Issue by ArthurZ** [oakes/SolidOak#39]
Copying, pasting from the clipboard (Ubuntu 14.04 LTS) does not seem to work.

Am I missing something?

**Comment by oakes** [comment]
I don't believe I got this working yet, unfortunately. We'll need to look at the [VteTerminal API](https://developer.gnome.org/vte/0.36/VteTerminal.html) to figure out how to enable it.

**Comment by ArthurZ** [comment]
In my view it is a critical item. 

Otherwise SolidOak is only good for an integrated execution of RUST code, not quite an effective editing. 

By the way 1st class Windows user support is also a necessity.

**Comment by skrzepto** [comment]
Seems like VteTerminal has paste from clipboard api call

https://developer.gnome.org/vte/0.36/VteTerminal.html#vte-terminal-paste-clipboard
